configuration/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@azure-iot/configuration",
+  "version": "0.0.1",
+  "description": "Provides configuration support for Azure IoT microservices",
+  "main": "configuration.ts",
+  "typings": "configuration.d.ts",
+  "author": "",
+  "dependencies": {
+    "mongodb": "^2.1.18",
+    "underscore": "^1.8.3"
+  },
+  "typings": "configuration.d.ts",
+  "scripts": {
```

Comment:
keep in mind that before the config library goes public you have to add this copyright to each file. We have a gulp task for this in common-ux common lib and dm-ux if you just want to copy the task from there

/* Copyright (c) Microsoft Corporation. All Rights Reserved. */

/*